llvm/test/CodeGen/AArch64/alias_mask_nosve.ll

```diff
@@ -0,0 +1,48 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
```

Comment:
nit: maybe merge this file with alias_mask_nosve2.ll and have two RUN lines?
(in that case, would expand_alias_mask.ll` be a better name?)